src/__experimental__/components/input/input.component.js

```diff
@@ -18,8 +18,9 @@ class Input extends React.Component {
     name: PropTypes.string,
     onBlur: PropTypes.func,
     onClick: PropTypes.func,
-    onFocus: PropTypes.func
-  }
+    onFocus: PropTypes.func,
+    type: PropTypes.string
+  };
 
   static contextType = InputPresentationContext
```

Comment:
Not sure how the linter hasn't picked this up but there's a couple of missing `;` in this file